functests/utils/profiles/profiles.go

```diff
@@ -25,12 +25,12 @@ func GetByNodeLabels(nodeLabels map[string]string) (*performancev1.PerformancePr
 	}
 
 	var result *performancev1.PerformanceProfile
-	for _, profile := range profiles.Items {
-		if reflect.DeepEqual(profile.Spec.NodeSelector, nodeLabels) {
+	for i := 0; i < len(profiles.Items); i++ {
```

Comment:
well, the broken code passed a review, so could you please add a couple lines to the commit message to emphasize why was broken before, so we know where to pay more attention next time?